file-upload-types.php

```diff
@@ -61,17 +63,18 @@ function file_upload_types_deactivate_msg() {
 define( 'FILE_UPLOAD_TYPES_VERSION', '1.0.0' );
 
 /**
- * Return the main instance of File_Upload_Types.
+ * Return the main instance of Plugin class.
  *
  * @since 1.0.0
  *
- * @return File_Upload_Types
+ * @return Plugin
```

Comment:
Technically, this function returns `\FileUploadTypes\Plugin` instance. The code in this file is not namespaced, so you need to define the FQCN here.